packages/next/src/server/lib/router-utils/setup-dev-bundler.ts

```diff
@@ -947,127 +929,24 @@ async function startWatcher(opts: SetupOpts) {
     return { finished: false }
   }
 
-  async function logErrorWithOriginalStack(
+  function logErrorWithOriginalStack(
```

Comment:
At this point it's really just a prefixed log. The only meaningful logic is silencing of TurbopackInternalError but that's a debateable strategy.

Might get rid of the prefixing or replace it with `console` instrumentation instead.